src/main/java/com/microsoft/sqlserver/jdbc/SQLServerDatabaseMetaData.java

```diff
@@ -675,6 +680,46 @@ private static String EscapeIDName(String inID) throws SQLServerException {
              * when user provides a different catalog than the one they're currently connected to. Will throw exception
              * when it's different and do nothing if it's the same/null.
              */
+            if (null == getColumnsDWColumns) {
```

Comment:
Please rename this map, there will be other APIs that require different column maps, so better the map name reflects the API name.

Reply:
it's meant to reflect the method where it's used. This is for `getColumns`, if `getImportedKeys` needs one it'd be `getImportedKeysDWColumns`.